addons/website/static/tests/builder/options/floating_blocks_option.test.js

```diff
@@ -0,0 +1,93 @@
+import { defineWebsiteModels, setupWebsiteBuilder } from "../website_helpers";
+import { expect, test } from "@odoo/hoot";
+import { contains, onRpc } from "@web/../tests/web_test_helpers";
+
+defineWebsiteModels();
+
+const websiteContent = `
+    <section class="s_floating_blocks pt32 pb32 o_cc o_cc2">
+        <div class="container">
+            <div class="d-flex flex-column gap-5">
+                <div class="s_floating_blocks_wrapper s_floating_blocks_wrapper_shadow d-contents rounded-4">
+                    <section
+                        data-name="Card"
+                        class="s_floating_blocks_block s_col_no_resize position-sticky parallax py-5 py-lg-0 d-flex o_cc o_cc5"
+                        data-scroll-background-ratio="-2"
+                        >
+                        <span class="s_parallax_bg oe_img_bg o_bg_img_center" style="background-image: url('/web/image/website.s_floating_blocks_1'); background-position: top center"/>
+                        <div class="o_we_bg_filter bg-black-25"/>
+                        <div class="container-fluid align-self-start align-self-lg-center">
+                            <div class="s_floating_blocks_block_grid oe_unremovable row mx-0 o_grid_mode" data-row-count="8">
+                                <div class="o_grid_item g-height-4 g-col-lg-8 col-lg-8 order-lg-0" style="z-index: 1; grid-area: 1 / 1 / 5 / 9;">
+                                    <p class="lead">Sofas made to last.</p>
+                                    <h2 class="display-4-fs">Jump Into <br class="d-none d-lg-inline"/>
+                                        Comfort!
+                                    </h2>
+                                </div>
+                                <div class="o_grid_item g-height-2 g-col-lg-4 col-lg-4" style="z-index: 2; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px; grid-area: 7 / 1 / 9 / 5;">
```

Comment:
you can use insertStructureSnippet or your use setupWebsiteBuilderWithSnippet(SNIPET_NAME)